### PR TITLE
Add documentation for Todoist component

### PR DESCRIPTION
**Pull request by Jay2645**
**Description:**
This is the documentation for the [Todoist](https://todoist.com) component that I've been working on.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#9236

### CODE REVIEW

File: source/_components/calendar.todoist.markdown [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    token: API_token_created_from_steps_above
+```
+
+Configuration variables:
+
+- **token** (*Required*): The API token used to authorize Home Assistant to access your projects.
+- **custom_projects** (*Optional*): Details on any "custom" binary sensor projects you want to create.
+  - **name** (*Required*): The name of your custom project. Only required if you specify that you want to create a custom project.
+  - **due_date_days** (*Optional*): Only include tasks due within this many days. If you don't have any tasks with a due date set, this returns nothing.
+  - **labels** (*Optional*): Any labels you want to filter for. All tasks in this sensor will include this label.
```
Comment (frenck) [inline source/_components/calendar.todoist.markdown]: **Spelling & Grammar**:
- `for` needs to go. -> `Any labels you want to filter.` 
Alternatively, you could replace the whole description with: `Only include tasks which have this label`

**Unclear documentation**:
- If I set two labels, will the tasks actually need to have both? (is it AND or OR?)

File: source/_components/calendar.todoist.markdown [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+As you can see, there are 4 custom projects here:
+
+- A project containing *all* of the tasks on this account.
+
+- A project containing *all* the tasks on this account that are due today.
+
+- A project containing *all* the tasks on this account due within the next week.
+
+- A projct containing everything with the label "Homework", taking only 2 projects into account.
```
Comment (frenck) [inline source/_components/calendar.todoist.markdown]: **Spelling & Grammar**:
- `projct`-> `project`

File: source/_components/calendar.todoist.markdown [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+- **task_labels**: All labels associated with this task.
+
+- **overdue**: Whether the reported task is past its due date.
+
+- **due_today**: Whether the reported task is due today.
+
+### {% linkable_title Services %}
+
+Todoist also comes with access to a service, `todoist.new_task`. This service can be used to create a new Todoist task. You can specify labels and a project, or you can leave them blank and the task will go to your "Inbox" project.
```
Comment (frenck) [inline source/_components/calendar.todoist.markdown]: **Spelling & Grammar**:
- Missing comma after `blank` -> `or you can leave them blank, and the task`

File: source/_components/calendar.todoist.markdown [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+}
+```
+
+- **content** (*Required*): The name of the task you want to create.
+
+- **project** (*Optional*): The project to put the task in.
+
+- **labels** (*Optional*): Any labels you want to add to the task, separated by commas.
+
+- **priority** (*Optional*): The priority of the task, from 1-4. Again, 1 means least important and 4 means most important.
```
Comment (frenck) [inline source/_components/calendar.todoist.markdown]: **Spelling & Grammar**:
- Missing coma after `important` -> `means least important, and 4 means`

File: source/_components/calendar.todoist.markdown [hunk 5]
```diff
@@ -0,0 +1,136 @@
+---
+layout: page
+title: "Todoist"
+description: "Instructions how to integrate Todoist into Home Assistant."
```
Comment (frenck) [inline source/_components/calendar.todoist.markdown]: **Spelling & Grammar**:
- `Instructions how to integrate` -> `Instructions on how to integrate`

**Comment by Jay2645** [comment]
@frenck That's what I get for writing documentation at 3 AM. Your changes should be live now.

**Comment by frenck** [comment]
@Jay2645 hehe, that sounds familiar 🤣 
Thanks for fixing it! 🥇

**Pull request merged**